src/style.css

```diff
@@ -1 +1,73 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+.page {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.search-container {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.search-container__search-bar {
+  width: 100%;
+}
+
+.search-bar__input {
+  width: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  padding: 0;
+}
+
+.search-bar__input--big {
+  font-size: 16px;
+  height: 70px;
+  line-height: 70px;
+  background-position: 26px center;
+  background-size: 19px;
+  margin: 20px 0;
```

Comment:
Be consistent with your margins to avoid potential margin collapse. It's recommended to add margins to the bottom of elements, rather than top and bottom. Adjust the margin to either `margin-bottom` or `margin-top` as needed throughout your design.